chainio/clients/elcontracts/writer_test.go

```diff
@@ -0,0 +1,225 @@
+package elcontracts_test
+
+import (
+	"context"
+	"log/slog"
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients"
+	rewardscoordinator "github.com/Layr-Labs/eigensdk-go/contracts/bindings/IRewardsCoordinator"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/testutils"
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRegisterOperator(t *testing.T) {
```

Comment:
similar comments for tests as in https://github.com/Layr-Labs/eigensdk-go/pull/360

would prefer test config + helper function for the anvil state setup